einterfaces/cluster.go

```diff
@@ -30,3 +31,85 @@ type ClusterInterface interface {
 	GetPluginStatuses() (model.PluginStatuses, *model.AppError)
 	ConfigChanged(previousConfig *model.Config, newConfig *model.Config, sendToOtherServer bool) *model.AppError
 }
+
+type ClusterImpl struct {
+}
+
+func NewClusterImpl() *ClusterImpl {
+	return &ClusterImpl{}
+}
+
+func (c *ClusterImpl) StartInterNodeCommunication() {
+	mlog.Error("StartInterNodeCommunication")
```

Comment:
Gentle reminder to check our logging [principles](https://developers.mattermost.com/contribute/server/style-guide/#log-levels) before merging this change.